Projects/Faker/Faker/Program.cs

```diff
@@ -1,63 +1,72 @@
 ﻿using System;
-using Faker;
-using Faker.Resources;
-using Newtonsoft.Json;
+using System.Linq;
+using System.Text.Json;
 
-namespace FakerDemo
-{    
-    internal class Program
+for (var i = 0; i < 25; i++)
+{
+    var user = CreateUser();
+
+    var userString = user.ToString();
+
+    // or, if you want JSON:
+    // var userString = JsonSerializer.Serialize(user, new JsonSerializerOptions {WriteIndented = true})
+
+    Console.WriteLine(userString);
+}
+
+User CreateUser()
+{
+    var fullNameInput = Faker.Name.FullName();
+
+    var titles = new[]
```

Comment:
Is this creating an empty array variable?

Reply:
It called a [collection initializer](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/object-and-collection-initializers#collection-initializers). It creates the the array and the values are set on line  22-29.